src/System.Windows.Extensions/src/System/Drawing/ImageConverter.cs

```diff
@@ -63,5 +75,75 @@ public override PropertyDescriptorCollection GetProperties(ITypeDescriptorContex
         }
 
         public override bool GetPropertiesSupported(ITypeDescriptorContext context) => true;
+
+        private unsafe Stream GetBitmapStream(byte[] rawData)
+        {
+            Debug.Assert(rawData != null, "rawData is null.");
+            try
+            {
+                fixed (byte* pByte = rawData)
+                {
+                    IntPtr addr = (IntPtr)pByte;
+
+                    if (addr == IntPtr.Zero)
```

Comment:
This is a weird way to check for null. We should just bail before we try and fix it, but outside of that I think this is an easy rewrite as Span.